### PR TITLE
test(dm): fix unstable validator case

### LINKED ISSUE #5730: validation status test new\/ignored\/resolved: 1\/0\/0 count: 0 != expected: 1
### Which jobs are flaking?

validator_basic

### Which test(s) are flaking?

DM-IT
release-6.1

### Jenkins logs or GitHub Actions link

```log
[2022-06-01T12:44:23.324Z] command: validation status test new\/ignored\/resolved: 1\/0\/0 count: 0 != expected: 1, failed the 9-th time, will retry again

[2022-06-01T12:44:25.220Z] {

[2022-06-01T12:44:25.220Z]     "result": true,

[2022-06-01T12:44:25.220Z]     "msg": "",

[2022-06-01T12:44:25.220Z]     "validators": [

[2022-06-01T12:44:25.220Z]         {

[2022-06-01T12:44:25.220Z]             "task": "test",

[2022-06-01T12:44:25.220Z]             "source": "mysql-replica-01",

[2022-06-01T12:44:25.220Z]             "stage": "Running",

[2022-06-01T12:44:25.220Z]             "validatorBinlog": "(dm-integration-test-go1180-6422-lw0nb-fj74q-bin.000001, 2590)",

[2022-06-01T12:44:25.220Z]             "validatorBinlogGtid": "",

[2022-06-01T12:44:25.220Z]             "result": null,

[2022-06-01T12:44:25.220Z]             "processedRowsStatus": "insert/update/delete: 6/1/1",

[2022-06-01T12:44:25.220Z]             "pendingRowsStatus": "insert/update/delete: 0/0/0",

[2022-06-01T12:44:25.220Z]             "allErrorRowsStatus": "new/ignored/resolved: 2/0/0"
```

https://ci2.pingcap.net/blue/organizations/jenkins/dm_ghpr_integration_test/detail/dm_ghpr_integration_test/6422/pipeline/

### Anything else we need to know

- Does this test exist for other branches as well?

- Has there been a high frequency of failure lately?

### PR DESCRIPTION
**Pull request by D3Hunter**
### What problem does this PR solve?

Issue Number: close #5730

### What is changed and how it works?
- increase row-error-delay to 1s to reduce the chance of unstable.

#### Tests 

 - Unit test
 - Integration test
 - Manual test (add detailed scripts or steps below)
 - No code

### Release note 

```release-note
`None`.
```

### CODE REVIEW

File: dm/tests/validator_basic/run.sh [hunk 1]
```diff
@@ -34,6 +34,7 @@ function prepare_for_standalone_test() {
 }
 
 function trigger_checkpoint_flush() {
+	sleep 1.5
```
Comment (Ehco1996) [inline dm/tests/validator_basic/run.sh:37]: the interval to flush validator checkpoint is 1s in this case right now, so need sleep before ddl(will trigger flush) for some time. below case is 1ms(will change to 1s in another fix pr) so sleep 0.1.

**Review by Ehco1996 — APPROVED** [review]

**Review by lance6716 — APPROVED** [review]

**Pull request merged**